### PR TITLE
Refactor randomization to have const input strings

### PR DESCRIPTION
**Pull request by dmcilvaney**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

The randomization class requires the caller to pass a class of characters to build random strings with, we should just have a set of well-known consts for these sets.

###### Change Log  

- Move `randomization.RandomString()` `legalCharacters` strings into the class instead of requiring the callers to redefine them.

###### Does this affect the toolchain?  

**NO**

###### Test Methodology

- local build

### CODE REVIEW

**Review by cwize1 — APPROVED** [review]

**Comment by cwize1** [comment]
Are you intentionally going into the `2.0` branch?

**Comment by dmcilvaney** [comment]
> Are you intentionally going into the `2.0` branch?

Nope, don't write PRs under the affects of cold medication! :P

**Review by mfrw — APPROVED** [review]
LGTM.

[digression]
Although [this](https://stackoverflow.com/a/31832326) [impl 7] does not use `crypto/rand`, I like how elegantly it is done.

File: toolkit/tools/imagegen/attendedinstaller/uitext/uitext.go [hunk 1]
```diff
@@ -156,8 +156,3 @@ const (
 	FinishTitle   = "CBL-Mariner Installation Complete"
 	FinishTextFmt = "Total installation time: %v seconds. Press Enter to restart."
 )
-
-// Common for input validation.
-const (
-	AlphaNumeric = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
```
Comment (mfrw) [inline toolkit/tools/imagegen/attendedinstaller/uitext/uitext.go:162]: I think there might a place that still references this:
`imagegen/attendedinstaller/views/diskview/manualpartitionwidget/manualpartitionwidget.go:657:111: undefined: uitext.AlphaNumeric`

**Pull request merged**